web/src/message_edit.ts

```diff
@@ -1252,6 +1252,30 @@ export function maybe_show_edit($row: JQuery, id: number): void {
     }
 }
 
+function warn_user_about_unread_msgs(last_sent_msg_id: number): void {
+    feedback_widget.show({
+        populate(container) {
+            container.html(
+                $t({
+                    defaultMessage:
+                        "Editing the last message you sent will mark all older messages in this view as read.",
+                }),
+            );
+        },
+        on_undo() {
+            // Select the message we want to edit to mark messages between it and the
+            // current selected id as read.
+            message_lists.current?.select_id(last_sent_msg_id, {
+                then_scroll: true,
+                from_scroll: true,
```

Comment:
Why are both of these true? I would expect `then_scroll` but not `from_scroll`...